scalafix-cli/src/main/scala-3/scalafix/internal/v1/TPrintImplicits.scala

```diff
@@ -0,0 +1,39 @@
+package scalafix.internal.v1
```

Comment:
copied from the `scala-2.12+` folder

Reply:
What about including [scala-2.12+ folders](https://github.com/scalacenter/scalafix/blob/c31ffa416c3ab4241e0849dcfd204b2e4b32336e/project/ScalafixBuild.scala#L207) for scala 3? there might be cases where it's not what we want, but if it works for now, I'd favor that over duplicating a file.